ipi/inputs/motion/motion.py

```diff
@@ -107,6 +107,7 @@ class InputMotionBase(Input):
                     "dummy",
                     "scp",
                     "normalmodes",
+                    "BEC",
```

Comment:
Why uppercase when it's lowercase everywhere?

Reply:
I just removed `BEC` from this file. It was useless.